test/wh_test_crypto.c

```diff
@@ -67,6 +67,16 @@ enum {
 
 #define PLAINTEXT "mytextisbigplain"
 
+/* Array allowing you to loop over available devIds */
+/* TODO: find a better way to do this - perhaps best to define a global LUT
+ * holding the devIds and refer to them by enumerated index rather than directly
+ */
+#ifdef WOLFHSM_CFG_DMA
+static const int DEV_IDS[WH_NUM_DEVIDS] = {WH_DEV_ID, WH_DEV_ID_DMA};
```

Comment:
why not something like:
static const int DEV_IDS[] = {x,a,b,};
#define WH_NUM_DEVIDS (sizeof(DEV_IDS) / sizeof(int))

devid == 0 is a valid id

Reply:
good call